src/pages/authors/AuthorsForm.vue

```diff
@@ -28,13 +31,60 @@ export default {
     }
   },
 
+  computed: {
+    isCreateAuthors () {
+      return this.$route.name === 'AuthorsCreate'
+    },
+
+    submitTitleLabel () {
+      return `${this.isCreateAuthors ? 'Criação' : 'Edição'} de autores`
+    },
+
+    submitButtonLabel () {
+      return this.isCreateAuthors ? 'Criar' : 'Editar'
+    },
+
+    idAuthor () {
+      return this.$route.params.id
+    }
+  },
+
+  created () {
+    if (!this.isCreateAuthors) {
+      this.setInputEditValues()
+    }
+    this.fetchAuthors()
```

Comment:
não entendi essa validação, se for edit ela usa o `fetchAuthor`, se não ela usa `fetchAuthors`, qual a diferença?

Reply:
o fetchAuthor eu estou pegando os autores pelo ID, para eu editar vou precisar só de um autor, ai estou fazendo isso pelo fetchAuthor, o fetchAuthors pega todos os autores

Reply:
mas pq vc precisa buscar todos autores na página de formulário?

Reply:
na verdade não precisa, eu estava com um raciocínio diferente, mas não faz sentido, retirado.